e2e_test/hawk_test_driver.py

```diff
@@ -329,7 +329,7 @@ def test_remove_cluster(self, cluster_name):
                 print("ERROR: Couldn't find cluster [%s]. Cannot remove" % cluster_name)
                 return False
             elem.click()
-        time.sleep(2 * self.timeout_scale)
+        time.sleep(6)
```

Comment:
this `6` is a  magic number.
As an improvement you should consider to introduce a constant like

`BIG_TIMEOUT` whatever fits.

Se also line 347. Why do you increase here the timeout and not on 347 for example?

Reply:
> Se also line 347. Why do you increase here the timeout and not on line 347 for example?

Removing cluster takes more time.

Reply:
Fixed in last commit